src/funcs/transformTableStructureToDBML.spec.js

```diff
@@ -0,0 +1,57 @@
+const  { getColumnType, getColumnDefault} = require('./transformTableStructureToDBML');
+
+describe('transformTableStructureToDBML && funcs used inside of it.', () => {
+  describe('getColumnType()', () => {
+    it('returns same data_type as given.', () => {
+      const columnType = { data_type: 'fakeColumnType'}
+      const response = getColumnType(columnType);
+      expect(response).toEqual(columnType.data_type)
+    })
+    it(`returns 'varchar' when data_type = 'character varying'.`, () => {
+      const columnType = { data_type: 'character varying'}
+      const response = getColumnType(columnType);
+      expect(response).toEqual('varchar')
+    })
+    it(`returns 'number' when data_type = 'double precision'.`, () => {
+      const columnType = { data_type: 'double precision'}
+      const response = getColumnType(columnType);
+      expect(response).toEqual('number')
+    })
+
+    it(`returns 'number' when data_type = 'timestamp with time zone'.`, () => {
+      const columnType = { data_type: 'timestamp with time zone'}
+      const response = getColumnType(columnType);
+      expect(response).toEqual('timestamp')
+    })
+    it(`returns 'number' when data_type = 'timestamp without time zone'.`, () => {
+      const columnType = { data_type: 'timestamp without time zone'}
+      const response = getColumnType(columnType);
+      expect(response).toEqual('timestamp')
+    })
+    it(`returns 'number' when data_type = 'time without time zone'.`, () => {
+      const columnType = { data_type: 'time without time zone'}
+      const response = getColumnType(columnType);
+      expect(response).toEqual('timestamp')
+    })
+    it(`returns undefined if not given coreect object input`, () => {
```

Comment:
```suggestion
    it(`returns undefined if not given correct object input`, () => {
```

Reply:
removed tests as they're a part of HPAPI-342.